ui-spacetimechart/src/stories/performances.stories.tsx

```diff
@@ -0,0 +1,188 @@
+import cx from 'classnames';
+import React, { FC, useMemo, useState } from 'react';
+import type { Meta } from '@storybook/react';
+
+import { getPaths, PATHS } from './assets/paths';
+import { SpaceTimeChart, PathLayer } from '../';
+import { X_ZOOM_LEVEL, Y_ZOOM_LEVEL, zoom } from './utils';
+import { getDiff } from '../utils/vectors';
+import { OperationalPoint, Point } from '../core/types';
+
+import './tailwind-mockup.css';
+import { random, range } from 'lodash';
+import { KILOMETER, MINUTE } from '../core/consts';
+
+const DATE_OFFSET = +new Date('2024/01/01');
+const COLORS = [
+  '#9E8256',
+  '#FF362E',
+  '#FF8E3D',
+  '#FFBF00',
+  '#95C877',
+  '#78E6C5',
+  '#66C0F1',
+  '#526CE8',
+  '#D16DBC',
+  '#FF9BC6',
+];
+
+/**
+ * This story aims at showcasing how to handle panning and zooming in a SpaceTimeChart.
```

Comment:
Description is not correct, this story aims at testing the performances of the chart when rendering a high number of paths.